src/test/kotlin/com/igorwojda/list/countuniquevalues/challenge.kt

```diff
@@ -4,7 +4,16 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUniqueValues(list: List<Int>): Int {
-    TODO("not implemented")
+    if (list.isEmpty())
```

Comment:
Each type of collection has its unique characteristics. What type of collection should you use and how, to guarantee that you only get distinct items?
https://kotlinlang.org/docs/collections-overview.html